tests/datasource/fluent/test_pandas_s3_datasource.py

```diff
@@ -46,34 +44,11 @@
 ]
 
 
-@pytest.fixture()
-def aws_region_name() -> str:
-    return "us-east-1"
-
-
 @pytest.fixture()
 def aws_s3_bucket_name() -> str:
     return "test_bucket"
 
 
-@pytest.fixture(scope="function")
-def aws_credentials() -> None:
-    """Mocked AWS Credentials for moto."""
-    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
-    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
-    os.environ["AWS_SECURITY_TOKEN"] = "testing"
-    os.environ["AWS_SESSION_TOKEN"] = "testing"
-    os.environ["AWS_DEFAULT_REGION"] = "testing"
```

Comment:
This is a dangerous fixture as these persist for the lifetime of the test.
I've moved this to a `conftest.py` file and used `pyest.monkeypatch.setenv()` so that this is unset at the end of the test.

It's possible we've had some tests that were relying on this state.